### ISSUE TITLE
debugFeatures crashes on opening debug tab

### ISSUE THREAD

**Issue by inphobia** [microsoft/terminal#12290]
### Windows Terminal version

1.11.3471.0

### Windows build number

19043.1415

### Other Software

not relevant, crashing on opening any tab.

### Steps to reproduce

* enable `"debugFeatures": true,` via edeting the config json
* restart windows terminal to activate it
* open a tab while holding left and right alt (on my keyboard - azerty be period layout, right alt is altgr no idea if that matters)

does not matter if the tab you open is from the dropdown list or via the "+" button. opening it via the shortcut ctrl+shift+t does nothing.

### Expected Behavior

openes a new tab with a debug window. or at least does not crash.

it wasn't quite clear if release builds of wt can even enable this feature.

### Actual Behavior

hangs for a few seconds, the crashes wt and all open tabs. (program just closs).

does work in current preview of wt.

eventlog shows this, dunno if there's any other debugging output to provide:
```
Faulting application name: WindowsTerminal.exe, version: 1.11.2112.13011, time stamp: 0x61b80d0f
Faulting module name: Microsoft.Terminal.Control.dll, version: 1.11.2112.13011, time stamp: 0x61b80b7f
Exception code: 0xc0000005
Fault offset: 0x000000000000fa8e
Faulting process id: 0x6674
Faulting application start time: 0x01d81582c31ccf1f
Faulting application path: C:\Program Files\WindowsApps\Microsoft.WindowsTerminal_1.11.3471.0_x64__8wekyb3d8bbwe\WindowsTerminal.exe
Faulting module path: C:\Program Files\WindowsApps\Microsoft.WindowsTerminal_1.11.3471.0_x64__8wekyb3d8bbwe\Microsoft.Terminal.Control.dll
Report Id: 698fd48e-3ac8-4f83-8a76-294d506043c9
Faulting package full name: Microsoft.WindowsTerminal_1.11.3471.0_x64__8wekyb3d8bbwe
Faulting package-relative application ID: App
```

**Comment by inphobia** [comment]
site note, also tested it with no `"debugFeatures": true,`

then alt left+right & mouseclick just open a new sidetab.

**Comment by elsaco** [comment]
@inphobia `"debugFeatures": true` is a hit-and-miss with each release. It works with the `Windows Terminal Preview Version: 1.12.3472.0`

![wt_debug_true](https://user-images.githubusercontent.com/3933920/151713215-5dd52b1a-469f-435f-acb1-cafb9ddedf89.png)

The only issue is debug pane not closing when exiting the tab, even though it exits with code zero:

![wt_debug_exit](https://user-images.githubusercontent.com/3933920/151713250-c0d82a83-79da-4708-84f1-6ee762665b8a.png)

It could be by design!

**Comment by zadjii-msft** [comment]
Yea this was fixed in #11445, which is in v1.12.2922.0+. /dup #11282
